src/containers/AdminCampaignList.jsx

```diff
@@ -133,11 +157,41 @@ class AdminCampaignList extends React.Component {
       });
   };
 
-  renderFilters() {
+  // handle selecting and de-selecting campaigns via CampaignListMenu
+  handleSelectForExport = (incomingCampaign) => {
+    const { campaignDetailsForExport: currentDetails } = this.state;
+    const currentIds = currentDetails.map((campaign) => campaign.id);
+    const isDeSelecting = currentIds.includes(incomingCampaign.id);
```

Comment:
Since these props get plumbed through a few layers of React components, I decided to handle `select` vs `de-select` logic within the handler - if you'd prefer, I can push the logic down to the `CampaignListRow` and pass down a `handleDeSelect` function as well

Reply:
that makes sense! i thought this might've been out of necessity rather than preference so good to know both approaches would work